python/servo/command_base.py

```diff
@@ -1059,7 +1078,12 @@ def find_highest_msvc_version():
             vsinstalldir = os.path.join(base_vs_path, version, edition)
             if os.path.exists(vsinstalldir):
                 return (vsinstalldir, vs_version, msbuild_version)
-    print("Can't find MSBuild.exe installation under %s." % base_vs_path)
+
+    vers = find_highest_msvc_version_ext()
+    for version in sorted(vers, key=lambda tup: float(tup[1])):
+        return version
```

Comment:
Let's write this as:
```python
versions = sorted(vers, key=lambda tup: float(tup[1]))
if not versions:
    print(...)
    sys.exit(1)
return versions[0]
```

Reply:
Should I replace vers here by the direct function call?

Reply:
I don't mind the way it is.